src/store/slices/ui.ts

```diff
@@ -3,21 +3,25 @@ import { onDrawerChange } from 'src/libs/url_utils';
 import { MenuType } from 'src/panel/Menu';
 import { GetState, State } from 'zustand';
 import { NamedSet } from 'zustand/middleware';
+import { modes } from 'src/adapters/direction_api';
 import { AppState } from '..';
 
 export interface UiSlice extends State {
   isMenuDrawerOpen: boolean;
   isProductsDrawerOpen: boolean;
   isSearchInputTyping: boolean;
+  defaultVehicle: string;
   setMenuDrawerOpen: (isOpen: boolean) => void;
   setProductsDrawerOpen: (isOpen: boolean) => void;
   setSearchInputTyping: (isSearchInputTyping: boolean) => void;
+  setDefaultVehicle: (defaultVehicle: string) => void;
 }
 
 export const createUiSlice = (set: NamedSet<AppState>, get: GetState<AppState>): UiSlice => ({
   isMenuDrawerOpen: false,
   isProductsDrawerOpen: false,
   isSearchInputTyping: false,
+  defaultVehicle: modes.WALKING,
```

Comment:
I think the `defaultVehicle` should be `modes.DRIVING` instead of `modes.WALKING`